plugins/ruby/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+# Allow build system to turn off downloading of libruby.so.
+OPTION(DOWNLOAD_RUBY "Download prebuilt libruby.so for ruby plugin." ON)
+
+IF (DOWNLOAD_RUBY)
+
```

Comment:
Done; I re-pushed the commit with everything between the initial IF(DOWNLOAD_RUBY) and ENDIF() indented one level.